adafruit_atecc/adafruit_atecc_asn1.py

```diff
@@ -126,15 +154,24 @@ def get_name(name, obj_type, data):
     return len(name) + 11
 
 
-def get_version(data):
-    """Appends X.509 version to data."""
+def get_version(data: bytearray):
```

Comment:
this can have the return type `-> None`

Although arguably it should be renamed since it doesn't seem to "get" anything and people calling this function might be expecting a return based on it's name. That is a separate issue from the typing effort though.